pkg/eks/update.go

```diff
@@ -162,6 +162,31 @@ func (c *ClusterProvider) UpdateClusterVersion(cfg *api.ClusterConfig) (*awseks.
 	return output.Update, nil
 }
 
+// UpdateClusterTags calls eks.TagResource and tags the cluster
+func (c *ClusterProvider) UpdateClusterTags(clusterConfig *api.ClusterConfig) error {
+	tags := make(map[string]*string)
+	for key, value := range clusterConfig.Metadata.Tags {
+		tags[key] = &value
+	}
+	if err := c.RefreshClusterStatus(clusterConfig); err != nil {
+		return err
+	}
+	input := &awseks.TagResourceInput{
+		ResourceArn: c.Status.clusterInfo.cluster.Arn,
+		Tags:        tags,
+	}
+	_, err := c.Provider.EKS().TagResource(input)
```

Comment:
just curious if this will override any existing tags creatd by Cloudformation if there is any.

Reply:
it doesn't

Reply:
right, just check that EKS CF now didn't suppor tags, and no tag propagration happened i.e. EKS tags are empty after provision